awx/ui/src/components/HealthCheckButton/HealthCheckButton.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Plural, t } from '@lingui/macro';
+import { Button, DropdownItem, Tooltip } from '@patternfly/react-core';
+import { useKebabifiedMenu } from 'contexts/Kebabified';
+
+function HealthCheckButton({ isDisabled, onClick, selectedItems }) {
+  const { isKebabified } = useKebabifiedMenu();
+  const hopNodeSelected =
+    selectedItems.filter((instance) => instance.node_type === 'hop').length > 0;
```

Comment:
This is a good catch, but I actually need to pass the selectedItems.length to the plural here instead of the `hopeNodeSelected.length`. I've refactored this work a bit to hopefully make it more clean.